README.md

```diff
@@ -147,6 +167,17 @@ participants:
   # Example; el_extra_labels: {"ethereum-package.partition": "1"}
   el_extra_labels: {}
 
+  # A list of tolerations that will be passed to the EL client container
+  # Only works with Kubernetes
+  # Example: el_tolerations:
+  # - key: "key"
+  #   operator: "Equal"
+  #   value: "value"
+  #   effect: "NoSchedule"
+  #   tolerationSeconds: 3600
```

Comment:
should we snake case this to be Kurtosis compliant?

Reply:
this has to be kubernetes compliant no?

Reply:
haha, at the end its k8s compliant but `toleration_seconds` is more in line with the rest of the configuration than `tolerationSeconds`

Reply:
Done